explorer/frontend/scripts/build.sh

```diff
@@ -10,12 +10,14 @@ if [ -n "$NIX_SSL_CERT_FILE" ]; then
   export SSL_CERT_FILE=$NIX_SSL_CERT_FILE
 fi
 
-export ARGS="--bridge-path frontend/src/Generated/"
+ARGS="--bridge-path frontend/src/Generated/"
 
 set -xe
 set -v
 
 pushd $scriptDir/../..
+  echo EXPLORER_NIX_FILE=$EXPLORER_NIX_FILE
+  echo EXPLORER_EXECUTABLE=$EXPLORER_EXECUTABLE
```

Comment:
Explorer is not the most critical thing to ensure it's unchanged, but if @sectore has 5 mins to spare, can surely chime in 😉